okhttp/src/main/java/okhttp3/internal/Util.kt

```diff
@@ -138,14 +136,14 @@ fun nonEmptyIntersection(
   return false
 }
 
-fun hostHeader(url: HttpUrl, includeDefaultPort: Boolean): String {
-  val host = if (":" in url.host) {
-    "[${url.host}]"
+fun HttpUrl.toHostHeader(includeDefaultPort: Boolean = false): String {
```

Comment:
Tempted to drop the `to` prefix and just call this` hostHeader()`. I like `to` to be more or less reversible. (Or did I pattern match to far?)

Reply:
This is reversible, I just didn't do the other one yet. It's directly above this function. Let me convert it too in the same commit.

Reply:
Done